api/src/certification/results/domain/usecases/get-certification-result-for-parcoursup.js

```diff
@@ -14,34 +14,34 @@ import { MoreThanOneMatchingCertificationError } from '../errors.js';
  * @param {string} params.firstName
  * @param {string} params.birthdate - Format YYYY-MM-DD
  * @param {string} params.verificationCode
- * @param {CertificationRepository} params.certificationRepository
+ * @param {CertificationParcoursupRepository} params.certificationParcoursupRepository
  *
  * @returns {CertificationResult} matching candidate certification result
  * @throws {MoreThanOneMatchingCertificationError} in some cases (INE for example) there might be duplicates
  * @throws {NotFoundError} if no certification exists for this candidate
  **/
-export const getCertificationResult = async ({
+export const getCertificationResultForParcoursup = async ({
```

Comment:
On sait deja que on est dans des resultats de certif

```suggestion
export const getResultForParcoursup = async ({
```